src/libs/CurrencySymbolUtils.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'underscore';
+import * as NumberFormatUtils from './NumberFormatUtils';
+
+/**
+ * Get localized currency symbol for SO4217 Code
+ * @param {String} preferredLocale
+ * @param {String} currencyCode
+ * @return {String}
+ */
+function getLocalizedCurrencySymbol(preferredLocale, currencyCode) {
+    const parts = NumberFormatUtils.formatToParts(preferredLocale, 0, {
+        style: 'currency',
+        currency: currencyCode,
+    });
+    const currencySymbol = _.find(parts, part => part.type === 'currency').value;
+    return currencySymbol;
+}
+
+/**
+ * Is currency symbol to left
+ * @param {String} preferredLocale
+ * @param {String} currencyCode
+ * @return {Boolean}
+ */
+function isCurrencySymbolLTR(preferredLocale, currencyCode) {
+    const parts = NumberFormatUtils.formatToParts(preferredLocale, 0, {
+        style: 'currency',
+        currency: currencyCode,
+    });
+
+    // The first element of parts will be type: currency for all currency
+    // Where it starts with symbol and the other will have it at last
+    // If it is not the first, it must be at last
```

Comment:
This is really confusing. What are you trying to say here?

Reply:
```javascript
// locale en
const formatter = new Intl.NumberFormat('en', { style: 'currency', currency: 'USD' });

formatter.formatToParts(99.99);

// return value:
[
  {
    "type": "currency",
    "value": "$"
  },
  {
    "type": "integer",
    "value": "99"
  },
  {
    "type": "decimal",
    "value": "."
  },
  {
    "type": "fraction",
    "value": "99"
  }
]

// locale es
const formatter = new Intl.NumberFormat('es', { style: 'currency', currency: 'USD' });

formatter.formatToParts(99.99);

// return value:
[
  {
    "type": "integer",
    "value": "99"
  },
  {
    "type": "decimal",
    "value": ","
  },
  {
    "type": "fraction",
    "value": "99"
  },
  {
    "type": "literal",
    "value": " "
  },
  {
    "type": "currency",
    "value": "US$"
  }
]
```

Reply:
Still don't get it.

Reply:
`isCurrencySymbolLTR` function is used to calculate whether the currency symbol is to the left or right. 

`Intl.NumberFormat.format` can be used to localize amount in any currency but It does not return whether the symbol is to the left or right (returns a string with symbol at correct position). For this we use `Intl.NumberFormat.formatToParts` which instead returns an array containing its parts (can see above comments). The part related to currency has an attribute `type` `currency`.  If the symbol is at first `currency` part is the first part and if the symbol is at last the `currency` part is at last.

Reply:
I understood what the code is doing. I meant to improve the comment. It is really confusing. 
```suggestion
    // Currency is LTR when the first part is of currency type.
```

Reply:
Sorry for misunderstanding

Reply:
Thanks for your help. Updated